### PR TITLE
Add instructions on assigning projects to billing groups

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
# What changed, and why it matters
Adding an article with instructions on how to assign projects to billing groups as there were no step-by-step instructions for this and to make it clearer where this feature is located now that the UI navigation has changed.

### CODE REVIEW

File: docs/platform/howto/billing-assign-projects.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+#. Select the billing group you want to assign the project to.
+
+#. Click the **Projects** tab.
+
+#. Click **Assign projects**.
+
+#. Select the projects to assign to this billing group and click **Assign projects**.
+
+#. Click **Cancel** to close the dialog box. You will see the newly assigned projects listed in this billing group.
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:24]: This comment applies to the current PR and the other one covering articles on creating and managing a billing group: How about adding final section "Related reading" (or similar) to all of the billing-related articles and cross-link between the three articles (for creating BG, managing BG, and assigning project to BG).

File: docs/platform/howto/billing-assign-projects.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Assign projects to billing groups
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
+
+.. note::
+
+    Projects can only be assigned to one billing group. If a project is already assigned to one billing group and you assign it to another, it will be removed from the first group.
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:10]: Would get rid of future tenses (this comment applies to the whole PR)

File: docs/platform/howto/billing-assign-projects.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Assign projects to billing groups
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_.
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:6]: I'd keep these sentences in one paragraph and append information what the user can actually learn from this article.

File: docs/platform/howto/billing-assign-projects.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
+
+.. note::
+
+    Projects can only be assigned to one billing group. If a project is already assigned to one billing group and you assign it to another, it will be removed from the first group.
+
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:11]: Any prerequisites to this operation?

File: docs/platform/howto/billing-assign-projects.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+To assign projects to a billing group:
+
+#. In the project, click **Billing**.
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:14]: How about replacing "click" with "select"? (this comment applies to the whole PR).

File: docs/platform/howto/billing-assign-projects.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
+
+.. note::
+
+    Projects can only be assigned to one billing group. If a project is already assigned to one billing group and you assign it to another, it will be removed from the first group.
+
+To assign projects to a billing group:
```
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:12]: projects => a project ?

File: docs/platform/howto/billing-assign-projects.rst (hunk 1 above)
Comment (wojcik-dorota) [inline docs/platform/howto/billing-assign-projects.rst:24]: This comment applies to the current PR and the other one covering articles on creating and managing a billing group: How about adding final section "Related reading" (or similar) to all of the billing-related articles and cross-link between the three articles (for creating BG, managing BG, and assigning project to BG).

Comment (staceysalamon-aiven) [reply docs/platform/howto/billing-assign-projects.rst:24]: You can contact the console dev teams in Slack if you see UX issues. :)

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**